blog/20250415-jupytergis-sync/index.md

```diff
@@ -0,0 +1,49 @@
+---
+title: "JupyterGIS sync meeting 2025-04-15"
+description: |
+  A weekly gathering of JupyterGIS team to discuss our progress and help each other out.
+  Open to all, but this meeting is _short_ and task-focused, so there will not be time
+  for introductions. Please join a core meeting!
+date: "2025-04-15"
+image: "/assets/images/standup.jpg"
+author:
+  - name: "The GeoJupyter community"
+categories:
+  - "JupyterGIS notes"
+tags: [jupytergis-notes]
+---
+
+:::info {.callout-info}
+:mag: This meeting is _short_ and _task-focused_. Unfortunately, we won't have time to
+introduce every new person to the team. Please attend core community meetings ([see
+GeoJupyter calendar](https://geojupyter.org/calendar)) to meet the team and add your own
+agenda items, and/or
+[introduce yourself on Zulip](https://jupyter.zulipchat.com/#narrow/channel/471314-geojupyter/topic/Welcome)!
+:::
+
+# JupyterGIS sync meeting 2025-04-15
+
+Please add new agenda items under the `New agenda items` heading!
+
+- [Join us on Google Meet](https://meet.google.com/zhk-vygf-gke)
+  - [What time is the meeting in my time zone?](https://dateful.com/convert/utc?t=4pm)
+- [Previous meetings](https://geojupyter.org/blog/#category=JupyterGIS%20notes)
+
+
+## Attendees
+
+Your name / GitHub ID / affiliation
+
+* Nicolas / @brichet  / QuantStack
+* Gregory / @gjmooney / QuantStack
+* Matt / @mfisher87  / DSE
+
+
+## Status reports
+
+
+## Requests for help
+
+* https://github.com/geojupyter/jupytergis/pull/631
+* https://github.com/geojupyter/jupytergis/pull/630
+  * Nicolas: Require the user to pass a path in for now. Consider a future separate widget `GISSandbox`, with visual indicators that it's ephemeral, and perhaps a button to convert it to a `GISDocument` with a filename.
```

Comment:
```suggestion
  * Nicolas: Require the user to pass a path in for now. Consider a future separate widget `GISSandbox`, with visual indicators that it's ephemeral, and perhaps a button to export it to `.jGIS` file.
```

Still to avoid confusion if running again the Notebook.